doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -262,15 +262,16 @@ Clone the Helm installer directory using the following command:
 
 Navigate to the
 ``amazon-sagemaker-operator-for-k8s/hack/charts/installer`` folder. Edit
-the \ ``values.yaml`` file, which includes high-level parameters for the
-Chart. Replace the ARN here with the ARN for the OIDC-based role you’ve
+the \ ``rolebased/values.yaml`` file, which includes high-level parameters for the
+Chart. Replace the role ARN here with the Amazon Resource Name (ARN) for the OIDC-based role you’ve
 created. 
 
 Install the Helm Chart using the following command:
 
 ::
 
-    helm install rolebased/ --generate-name
+    kubectl create namespace sagemaker-k8s-operator-system
+    helm install --namespace sagemaker-k8s-operator-system sagemaker-operator rolebased/
```

Comment:
We should add a note that customers can pick their own namespace, but it must be specified in `"system:serviceaccount:sagemaker-k8s-operator-system:sagemaker-k8s-operator-default"` in the `trust.json` file. Otherwise, authentication won't work and customers will be confused.

Reply:
Great idea. Added a note directly underneath

Reply:
(optional) there are actually some fun directives you can use for formatting these kinds of things: https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#admonitions

Reply:
Is there a good way to check for syntax errors? (A make target?) I'd like to use the warning admonition

Reply:
yeah, you can build the docs locally: https://github.com/aws/sagemaker-python-sdk#building-sphinx-docs